Web/src/components/games/MentalSums/MentalSumsGame.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import seedrandom from 'seedrandom';
+import styled from 'styled-components';
+import { blink } from 'utils/animations';
+import { useQuestionStream } from './ProblemGen';
+
+interface MentalSumsGameProps {
+  incrementScore: () => void;
+  score: number;
+  timeLeft: number;
+  seed: string;
+}
+
+const GameContainer = styled.div`
+  background: var(--pale-yellow);
+  overflow: hidden;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  z-index: -999;
+
+  & > * {
+    z-index: 1;
+  }
+`;
+
+interface BackgroundContainerProps {
+  animate: boolean;
+}
+
+const CorrectContainer = styled.div`
+  animation: ${blink} 0.5s ease-in-out 0s 1;
+  position: fixed;
+  background: var(--green);
+  overflow: hidden;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  z-index: 1;
+  opacity: 0%;
+`;
+
+const WrongContainer = styled.div`
+  animation: ${blink} 0.5s ease-in-out 0s 1;
+  position: fixed;
+  background: var(--red);
+  overflow: hidden;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  z-index: 1;
+  opacity: 0%;
+`;
+
+const StyledInput = styled.input`
+  font-size: 3rem;
+  text-align: center;
+  background: none;
+  outline: none;
+  border: none;
```

Comment:
Actually can help me move this in to `index.css` instead because the default borders are so ugly so we probably won't ever need them :>>